analysis_options.yaml

```diff
@@ -25,3 +25,4 @@ linter:
     no_literal_bool_comparisons: true
     prefer_relative_imports: true
     unnecessary_statements: true
+    unawaited_futures: true
```

Comment:
The commit message steps into a classic GitHub pitfall :slightly_smiling_face: :

    Partially fixes: #731

GitHub doesn't understand the "partially", so that will close the issue.

If you don't mean "fixes", then the trick is to put some other word between it and the issue number. For example:
```
Fixes-partly: #731
```

Reply:
nit in commit message: the convention for email-header-style lines at the end of a commit message is that the words before the colon are joined by hyphens `-`, not spaces ` `